### ISSUE TITLE
Contact: Notes

### ISSUE THREAD

**Issue by palbert1976** [WhiteKarma/NalandaERP#16]
Somehow "food allergies" ended up as a note on the main contact page. This information would be more appropriate on the health page/tab and should be relatively close the the allergies to medicine question. It would never occur to me to check "notes if a I had medical emergency related to food allergies.

**Comment by WhiteKarma** [comment]
I understand that this information seems totally out of place. The idea was: As health information is going to be reserved ( naturally to some restricted profiles ) and food allergies, on the interest of the contact itself, should be available for everyone, i forced this organization. But we can discuss this in the next meeting. And of course, also here.

Thank you!!!

**Comment by LVS04** [comment]
I agree with palbert, it is much better to group information coherently. So, is it possible to solve your problem of confidentiality by assigning confidentiality level to a bloc instead of a page?

**Comment by WhiteKarma** [comment]
Yes it is possible.

Thanks

**Comment by LVS04** [comment]
I am happy for you!!! :)

 

De : WhiteKarma [mailto:notifications@github.com] 
Envoyé : lundi 3 octobre 2016 15:05
À : WhiteKarma/NalandaERP <NalandaERP@noreply.github.com>
Cc : LVS04 <lvs04@yahoo.fr>; Comment <comment@noreply.github.com>
Objet : Re: [WhiteKarma/NalandaERP] Contact: Notes (#16)

 

Yes it is possible.

Thanks
